fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -1009,8 +1009,13 @@ public Granularity getGranularity() {
         return this.intervals;
     }
 
+    public Map<Dimension, Set<ApiFilter>> getApiFilters() {
+        return this.apiFilters;
+    }
+
+    @Deprecated
     public Map<Dimension, Set<ApiFilter>> getFilters() {
```

Comment:
These deprecated methods should delegate to the non-deprecated version (i.e. `getFilters()` should return `getApiFilters()`. Furthermore, we should add a Javadoc, with a `@deprecated` annotation indicating which method to use instead.